{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -4,10 +4,10 @@
 # TODO: Pass the services required by your module to the
 # docker-services-cli e.g. `docker-services-cli up es postgresql redis`
 docker-services-cli up
-pydocstyle {{ cookiecutter.package_name }} tests docs && \
-isort {{ cookiecutter.package_name }} tests --check-only --diff && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
+python -m pydocstyle {{ cookiecutter.package_name }} tests docs && \
+python -m isort {{ cookiecutter.package_name }} tests --check-only --diff && \
+python -m check-manifest --ignore ".travis-*" && \
+python -m sphinx-build -qnNW docs docs/_build/html && \
```

Comment:
Note that some of the commands have to be expressed differently when using `python -m`; have a look at [this comment](https://github.com/inveniosoftware/invenio-db/pull/131#discussion_r476224349) for further details.

Reply:
Thanks @FlorianCassayre! Could we make some sort of recipe or common point where this is documented? I failed to find it before twice (my bad sorry about that).

Reply:
No problem of course, there are a lot of discussions spreading on different tickets. Good point, though I'm not sure where that should be documented...